Submissions/DEV-Soumya-Ranjan-Swain.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "[Soumya Ranjan Swain]",
```

Comment:
Please remove _all_ the square brackets, in the README.md they are meant to be placeholders